### PR TITLE
DataGrid / TreeList: Fix row focus after filtering (T993300)

### PR DESCRIPTION
**Pull request by Alyar666**
See https://devexpress.com/issue=T993300

### CODE REVIEW

File: js/ui/tree_list/ui.tree_list.focus.js [hunk 1]
```diff
@@ -92,8 +92,6 @@ core.registerModule('focus', extend(true, {}, focusModule, {
 
                     that.expandAscendants(key).done(function() {
                         dataSource.load({
-                            filter: that.getCombinedFilter(),
-                            sort: that.getController('columns').getSortDataSourceParameters(!dataSource.remoteOperations().sorting),
```
Comment (NickMitrokhin) [inline js/ui/tree_list/ui.tree_list.focus.js:96]: Why did you remove filter and sort?

Comment (Alyar666) [reply js/ui/tree_list/ui.tree_list.focus.js:96]: The TreeList takes filter and sort later when loading. So I removed this to avoid duplication

**Review by vconst — APPROVED** [review]

**Review by NickMitrokhin — APPROVED** [review]
